lib/matplotlib/backends/backend_wx.py

```diff
@@ -1023,20 +1026,31 @@ def _onKeyUp(self, evt):
         evt.Skip()
         FigureCanvasBase.key_release_event(self, key, guiEvent=evt)
 
+    def _set_capture(self, capture=True):
```

Comment:
I don't see it being called anywhere without a value, so maybe the default value isn't necessary.